ml-conversational-analytic-tool/featureVector.py

```diff
@@ -32,6 +32,7 @@ def readRawData(self, filename):
         """
         self.raw_filename = filename
         self.raw_data = pd.read_csv(filename)
+        self.raw_data.dropna(inplace=True)
```

Comment:
This change looks to cut around 30% from our data. 
In our raw data file there are rows with `nan` values for all PR's - Body, Comments, and Review_Comments. 
So, I am suggesting "normalizing" our data as in the following function we substitute all `nan` values with empty dir https://github.com/vmware-labs/ml-conversational-analytic-tool/blob/63c6620b9963f18cc22a0d74457bd44a3ba43273/ml-conversational-analytic-tool/utils.py#L39-L44
This should fix the case for Comments and review_comments
WDYT?
For PR's body We need just to fix a small bug that currently exists. Line 31 in the code bellow show use `cleaned_comment` instead of `comment
https://github.com/vmware-labs/ml-conversational-analytic-tool/blob/63c6620b9963f18cc22a0d74457bd44a3ba43273/ml-conversational-analytic-tool/commentAnalysis.py#L21-L31

Reply:
An empty list for comments and reviews sounds good to me.

As for the body, we shouldn't use `clean_comments` in line 31 due to [VADER recognizing capitalization emphasis](https://github.com/cjhutto/vaderSentiment#code-examples) in text. The sentiment score will change if we use cleaned text.

Reply:
Agree with your comment, then we need to at least  handle `nan` values